plugin/loader/initializer.go

```diff
@@ -41,3 +52,15 @@ func runIPLDPlugin(pl plugin.Plugin) error {
 
 	return ipldpl.RegisterInputEncParsers(coredag.DefaultInputEncParsers)
 }
+
+func runTracerPlugin(pl plugin.Plugin) error {
```

Comment:
since we're doing a type cast above, we can make this function just accept a `PluginTracer` and skip the cast here. Should do the same for the ipld one too